index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+    <title>Hammad's Portfolio </title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+    <div class="app-wrap">
+    <header>
+        <nav class="toolbar">
+            <div class="logo">
+                My Logo
+            
+            <a href="#" class="button">
+              <i class="fa fa-bars"></i>
+            </a> 
+        </div>
+        </nav>
+    </header>
+
+    <main class="page">
+        <section class="intro">
+            <h2 class="name">I'm Hammad</h2>
+            <h2 class="name"> Glad to see you!</h2>
+            <p class="description">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        </section>
```

Comment:
- [ ] kindly add social icons like the given design The icons should be in a `<ul>` tag to group them, and `<li>` per icon; each icon has a link (`<a>` tag)

![image](https://user-images.githubusercontent.com/68796232/158702371-127b1e8f-025a-468a-b03a-66a57acea9c6.png)